drivers/SmartThings/deepsmart/src/lifecycles.lua

```diff
@@ -19,19 +24,34 @@ function lifecycle_handler.init(driver, device)
     -- initialized.
     -- Refresh schedule
     log.info('device '..device.parent_assigned_child_key..' in lifecycle init')
+    -- refresh subdevice status
+    driver:inject_capability_command(device, {
+    capability = caps.refresh.ID,
+    command = caps.refresh.commands.refresh.NAME,
+    args = {}
+  })
   end
 end
 
 function lifecycle_handler.added(driver, device)
   log.info('device '..device.id..' added')
   if (device.parent_assigned_child_key == nil) then
+    log.info('bridge device '..device.id..' added')
+    -- for bridge save some infos to bridge device
+    device:set_field(config.FIELD.TYPE, "bridge", { persist = true })
```

Comment:
Since `FIELD.TYPE` is only one of two values set based on parent_assigned_child_key, I think we should check that field rather than adding another field to every device.

Reply:
`FIELD.UUID` also seems unnecessary since the information is already present on all device objects.